test_communication/package.xml

```diff
@@ -11,21 +11,23 @@
   <maintainer email="dthomas@osrfoundation.org">Dirk Thomas</maintainer>
   <license>Apache License 2.0</license>
 
+  <build_depend>ament_cmake_auto</build_depend>
+  <build_depend>builtin_interfaces</build_depend>
+  <build_depend>rclcpp</build_depend>
+  <build_depend>rclpy</build_depend>
+  <build_depend>rcl</build_depend>
```

Comment:
a2e014c reshuffles the depend tags as suggested, and the desired code generation still works. I learn something new every day.